modal_proto/api.proto

```diff
@@ -204,11 +204,22 @@ message BlobCreateRequest {
   // TODO(erikbern): how are these garbage collected?
   // Shouldn't they belong to an app?
   string content_md5 = 1;
+  string content_sha256_base64 = 2;
+  int64 content_length_bytes = 3;
```

Comment:
micro nit, but i think of "bytes" as a default unit for anything like this, so it could be omitted